spec/factories/users.rb

```diff
@@ -89,5 +93,15 @@
         full_name.downcase.gsub(' ', '.') + '@computacenter.com'
       end
     end
+
+    trait :computacenter_concerned do
+      has_seen_privacy_notice
+      orders_devices { true }
+    end
+
+    trait :not_computacenter_concerned do
+      has_not_seen_privacy_notice
+      orders_devices { false }
+    end
```

Comment:
I think the name of these traits could be a bit more clear - maybe `(not_)relevant_to_computacenter` or `(not_)for_computacenter` instead?